### PR TITLE
Clean up code sample blocks

### PR DESCRIPTION
**Pull request by davkean**
- Mark them as a specific language so they get syntax highlighting
- Unify spacing to 4 spaces
- Use … consistently

This is entirely containing only stylistic changes and no changes to the samples themselves.

Click the following button on the _Files changed_ tab to make this easier to see the changes.

image

### CODE REVIEW

**Comment by davkean** [comment]
ping @jimmyca15.

File: README.md [hunk 1]
```diff
@@ -76,38 +76,40 @@ The `FeatureManagement` section of the json document is used by convention to lo
...
-    {
-        FeatureT,
-        FeatureU,
-        FeatureV
-    }
+``` C#
+// Define feature flags in an enum
+public enum MyFeatureFlags
+{
+     FeatureT,
```
Comment (jimmyca15) [inline README.md]: Looks like there should be 1 less space for each of the enum values.

**Comment by jimmyca15** [comment]
Thanks @davkean, looks great with the syntax highlighting. Just one comment.

File: README.md [hunk 2]
```diff
@@ -76,38 +76,40 @@ The `FeatureManagement` section of the json document is used by convention to lo
...
+     FeatureU,
+     FeatureV
```
Comment (davkean) [inline README.md]:
```suggestion
    FeatureT,
    FeatureU,
    FeatureV

```

Comment (davkean) [inline README.md]:
```suggestion
    FeatureT,
    FeatureU,
    FeatureV
```

**Comment by davkean** [comment]
@jimmyca15 Fixed, feel free to squash to throw away that additional commit.

**Pull request merged**